buildSrc/src/main/kotlin/com.hedera.hashgraph.aggregate-reports.gradle.kts

```diff
@@ -18,7 +18,6 @@ import java.io.BufferedOutputStream
  */
 
 plugins {
-    `java-platform`
```

Comment:
This is a plugin for defining dependency versions through _dependency constraints_ – that is, you can use this to create (and possibly publish) a BOM. I think this was just misplaced here and had no effect.